setup.cfg

```diff
@@ -25,20 +25,14 @@ project_urls =
 [options]
 packages = 
     lsqecc
-python_requires = >=3.9
+python_requires = >=3.7
```

Comment:
I'm curious as to why we are adding support for python 3.7 and 3.8. Supporting more versions can become a big responsibility, it can generate a lot of work and restrict us from using new features.

I'd be very careful before adding support for older python version, and wouldn't do it without a good reason.

Reply:
After all this is a research project, not a foundational enterprise tool :sweat_smile: so I think we should try not to create work for us that isn't really serving anyone.

This might change if we have clients though, but the way this project is a cuting edge, I think functionality and results are the driving force.

Reply:
Our main version is still 3.9. Right now I am having that setting there to take advantage of Github Actions allowing us to run tests on different python version in parallel. Once problems arise 3.7 or 3.8 that we cannot resolve quickly we will remove support for that version.